java/src/main/java/com/company/automation/classes/main_task/Student.java

```diff
@@ -0,0 +1,168 @@
+package com.company.automation.classes.main_task;
+
+/*
+Создать классы, спецификации которых приведены ниже. Определить конструкторы и методы setТип(), getТип(), toString().
+Определить дополнительно методы в классе, создающем массив объектов. Задать критерий выбора данных и вывести эти данные на консоль.
+В каждом классе, обладающем информацией, должно быть объявлено несколько конструкторов.
+
+1. Student: id, Фамилия, Имя, Отчество, Дата рождения, Адрес, Телефон, Факультет, Курс, Группа.
+Создать массив объектов. Вывести:
++ a) список студентов заданного факультета;
++ b) списки студентов для каждого факультета и курса;
++ c) список студентов, родившихся после заданного года;
++ d) список учебной группы.
+
+ */
+
+import java.time.LocalDate;
+
+public class Student {
+    private int id;
+    private String firstName;
+    private String secondName;
+    private String additionalName;
+    private LocalDate birthDate;
+    private String address;
+    private String phoneNumber = "";
+    private int group;
+    private Course course;
+    private Faculty faculty;
+
+    public Student(int id, String firstName, String secondName, String additionalName, int year, int month, int day, String address) {
```

Comment:
You can use LocalDate as a constructor parameter and create the Student like:
Student student07 = new Student(64, "Lisa", "Simpson", "", LocalDate.of(1982, 3, 17), "USA, Springfield", "555-29-14", Course.SECOND, 122199, Faculty.INFO_COMMUNICATION);

Reply:
Fixed